src/modules/about-us/HeroSection/BannerImage.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Image from 'next/image';
+import aboutusbanner from "@/assets/about-us/about-us-banner.jpg";
```

Comment:
hmm, should we move them to `public/images`?

Reply:
Yeah. Will fix.